kitsune/journal/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+from kitsune.journal.models import Record
+
+
+class RecordAdmin(admin.ModelAdmin):
```

Comment:
I think this works. I am thinking that we could put a json string in the msg if we need to capture some sort of structured data. That can then be processed offline.